src/storage/BaseProcessor.h

```diff
@@ -65,9 +65,6 @@ class BaseProcessor {
 
     void doRemove(GraphSpaceID spaceId, PartitionID partId, std::vector<std::string> keys);
 
-    kvstore::ResultCode doRange(GraphSpaceID spaceId, PartitionID partId, std::string start,
```

Comment:
> Why remove it?

Because exists  as below, this is redundant. 

```
kvstore::ResultCode doRange(GraphSpaceID spaceId, PartitionID partId, const std::string& start,
                                const std::string& end, std::unique_ptr<kvstore::KVIterator>* iter);
```